scripts/prepare_eez_data/prepare_eez_data/test_eez_abs_preparation.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+import os
+from prepare_eez_abs_data import EezAbsMapper  # Replace 'mymodule' with the actual module name
+
+
+class TestEezAbsMapper(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        # Define the paths to the existing test shapefile and ABS data
+        cls.shapefile_path = 'test_data/eez_v12/eez_v12.shp'
+        cls.abs_csv_path = 'test_data/abs_info.csv'
+        cls.output_json_file = 'test_data/test_output.json'
```

Comment:
Unless these files are gigantic, I'd suggest including them in the repo (i.e. removing `test-data` from the gitignore) so it is easy to run this test.